third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/mutate_combo_invalid.exp

```diff
@@ -1,48 +1,22 @@
 
```

Comment:
Love that random error message ordering.  But https://github.com/aptos-labs/aptos-core/issues/10935 is already filed.

Nice that we've reduced the duplication.